lib/ProductOpener/APITest.pm

```diff
@@ -442,7 +442,15 @@ sub execute_api_tests ($file, $tests_ref, $ua = undef) {
 				);
 			}
 			elsif (defined $test_ref->{form}) {
-				$response = $test_ua->post($url, Content => $test_ref->{form}, %$headers_in);
```

Comment:
For POST requests, we can add Content_Type => 'multipart/form-data' to the post() call, and keep the same call for both forms that contain files (indicated by passing an array reference as the value of a parameter, like imgupload_front_en => ["/path/file.jpg"] ), and forms that don't contain files